docs/readmes/basics/prerequisites.md

```diff
@@ -66,13 +66,13 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
    1. Download the tar file.
 
       ```bash
-      wget https://artifactory.magmacore.org/artifactory/generic/go1.18.3.linux-amd64.tar.gz
+      wget https://artifactory.magmacore.org/artifactory/generic/go1.19.3.linux-amd64.tar.gz
```

Comment:
You also need to change this in the MacOS section further above (lines 34 and 37)

Reply:
Line 64 also mentions the golang version.